src/@ionic-native/plugins/purchases/index.ts

```diff
@@ -190,7 +190,7 @@ export enum INTRO_ELIGIBILITY_STATUS {
  * Please follow the [Quickstart Guide](https://docs.revenuecat.com/docs/) for more information on how to use the SDK
  *
  * ### Requirements
- * Requires XCode 10.2+ and minimum targets iOS 9.0+ and macOS 10.12+
+ * Requires XCode 10.2+ and minimum targets iOS 9.0+, macOS 10.12+, watchOS 6.2+, tvOS 9.0+
```

Comment:
not sure whether this is worth adding, given that ionic only supports iOS for now. 
However, it looks like it doesn't support Mac either, so it's _as_ valid as macOS I guess

Reply:
Also, don't we require the newer xcode now? I think we should just lleave iOS

Reply:
apple does, at least, starting this month, so yeah. I'll update to Xcode 11 and kill the other OS versions